pkg/apis/zookeeper/v1beta1/zookeepercluster_types.go

```diff
@@ -141,6 +141,12 @@ type ZookeeperClusterStatus struct {
 	ExternalClientEndpoint string `json:"externalClientEndpoint"`
 
 	MetaRootCreated bool `json:"metaRootCreated"`
+
+	// CurrentVersion is the current cluster version
+	CurrentVersion string `json:"currentVersion"`
+
+	// UpdateFlag indicate if update of the cluster is in progress or not
+	UpdateFlag bool `json:"updateFlag"`
```

Comment:
Instead of adding a flag it would be better to have `Status Conditions` added to ZK CRD and the Upgrading Condition set to true when Upgrade is in Progress.